jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -292,13 +292,13 @@ bool ci_lazy_methodst::handle_virtual_methods_with_no_callees(
     }
 
     // Check that `get_virtual_method_target` returns a method now
-    const irep_idt &call_basename = virtual_function.get_component_name();
-    const irep_idt method_name = get_virtual_method_target(
-      instantiated_classes, call_basename, call_class, symbol_table);
-    CHECK_RETURN(!method_name.empty());
+    const irep_idt &method_name = virtual_function.mangled_method_name();
```

Comment:
Might be easier if we used `method_base_name` and `mangled_method_base_name`, making it clear that neither includes the package/class qualifiers?

Reply:
To me, `base_name` means "not mangled". So `mangled_method_base_name` would read as a contradiction.